### PR TITLE
Mitigate pip CVE by using ubi9 9.3.

### PR DESCRIPTION
**Pull request by OliviaHY**
APPSRE-7764

### CODE REVIEW

File: qontract-reconcile-builder/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.5.2
+0.5.3
```
Comment (BumbleFeng) [inline qontract-reconcile-builder/VERSION:1]: Is there any change to the qontract-reconcile-builder?

Comment (OliviaHY) [reply qontract-reconcile-builder/VERSION:1]: No but if it's based on a different base, doesn't the version need to be bumped?

Comment (OliviaHY) [reply qontract-reconcile-builder/VERSION:1]: Actually the [tag](https://github.com/app-sre/container-images/blob/master/qontract-reconcile-builder/Dockerfile#L1) need to be updated to 0.10.3 but need to happen after it was built

Comment (OliviaHY) [reply qontract-reconcile-builder/VERSION]: I have removed the change for qontract-reconcile-builder

**Review by BumbleFeng — APPROVED** [review]

**Pull request merged**